Casks/font/font-u/font-urw-core35.rb

```diff
@@ -0,0 +1,88 @@
+cask "font-urw-core35" do
+  version "1.0.0"
+  sha256 "6cfedddd6733f99172778330b779d35e62360eebf6b7f36002ff65178ee21495"
+
+  url "https://github.com/twardoch/urw-core35-fonts/archive/refs/tags/v#{version}.tar.gz"
+  name "font-urw-core35"
+  desc "Fork of URW ++ Type 1 fonts https://github.com/URWTypeFoundry/Core_35"
```

Comment:
```suggestion
  name "URW Core 35 Fonts"
```

We also don't include `desc` stanzas for font casks anymore.